File: src/com/eagle/programmar/Template/Terminals/Template_KeywordChoice.java
// Copyright Eagle Legacy Modernization, LLC, 2010-date
// Original author: Steven A. O'Hara, Nov 12, 2015

package com.eagle.programmar.Template.Terminals;

import com.eagle.tokens.TerminalKeywordChoice;

public class Template_KeywordChoice extends TerminalKeywordChoice
{
	// Need default constructor for reading from the XML file
	public Template_KeywordChoice()
	{
		super();
	}
	
	public Template_KeywordChoice(String... words)
	{
		super(words);
	}
}

File: csrc/com/eagle/programmar/Template/Terminals/Template_KeywordChoice.cs
// ====================================================================================================
// Produced by the Free Edition of Java to C# Converter.
// Purchase a Premium Edition license at:
// https://www.tangiblesoftwaresolutions.com/order/order-java-to-csharp.html
// ====================================================================================================

// Copyright Eagle Legacy Modernization, LLC, 2010-date
// Original author: Steven A. O'Hara, Nov 12, 2015

namespace com.eagle.programmar.Template.Terminals
{
	using TerminalKeywordChoice = com.eagle.tokens.terminals.TerminalKeywordChoice;

	public class Template_KeywordChoice : TerminalKeywordChoice
	{
		// Need default constructor for reading from the XML file
		public Template_KeywordChoice() : base()
		{
		}

		public Template_KeywordChoice(params string[] words) : base(words)
		{
		}
	}

}
